catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -311,6 +311,12 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Metacard.RESOURCE_CACHE_STATUS,
+                false /* indexed */,
+                false /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                STRING_TYPE));
```

Comment:
Shouldn't this be BOOLEAN_TYPE? Seems like only true/false is being used or am I missing something?